core/src/test/java/org/bitcoinj/core/TransactionTest.java

```diff
@@ -147,7 +148,7 @@ public void coinbaseScriptSigTooLarge() {
     @Test
     public void testEstimatedLockTime_WhenParameterSignifiesBlockHeight() {
         int TEST_LOCK_TIME = 20;
-        Instant now = TimeUtils.currentTime();
+        Instant now = TimeUtils.currentTime().truncatedTo(ChronoUnit.SECONDS);
```

Comment:
If this use-case is common, you might want to consider a `TimeUtils.currentBlockTime()` which would return the time truncated to seconds.